Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/Controller.purs

```diff
@@ -798,14 +810,32 @@ eval (MessagingViewActionController (MessagingView.SendSuggestion chatSuggestion
     continue state {data {chatSuggestionsList = []}, props {canSendSuggestion = false}}
   else continue state
 
+eval (MessagingViewActionController (MessagingView.MultiChat)) state = do 
```

Comment:
Can we name this ShowMultiChatPopup or ShowMultiChat because here we are just showing the list of emergency contacts or driver so that customer can chat with them?